app/authentication/authenticator.py

```diff
@@ -160,14 +163,14 @@ def store_session(metadata):
     logger.info("user authenticated")
 
 
-def decrypt_token(encrypted_token):
+def decrypt_token(encrypted_token: str) -> dict[str, Any]:
```

Comment:
I wonder if this could be a bit tighter, I had a quick look at what is returned with a decrypted_token and it only looks like str, int and list